package.json

```diff
@@ -1,7 +1,12 @@
 {
   "name": "root",
   "private": true,
+  "scripts": {
+    "watch": "nodemon --exec make update"
```

Comment:
This feels hacky, but I like it 👍

Reply:
Super hacky, not sure we actually want to keep it but it does work. I think you're right that `nx` will solve this problem for us but I tried for a bit and couldn't get it working.